cloudformation.yaml

```diff
@@ -223,3 +223,23 @@ Resources:
       TimeToLiveSpecification:
         AttributeName: "ttlTimestamp"
         Enabled: true
+
+  LambdaThrottledAlarm:
+      Type: "AWS::CloudWatch::Alarm"
+      Properties:
+       AlarmActions:
+          - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:fulfilment-dev
+       AlarmDescription: !Sub "DigitalSubAuth-${Stage} has throttled invocations"
+       AlarmName: !Sub "DigitalSubAuth-${Stage}_Throttle"
+       Dimensions:
+        - Name: FunctionName
+          Value: !Sub DigitalSubAuth-${Stage}
+       MetricName: Throttles
+       Namespace: AWS/Lambda
+       TreatMissingData: notBreaching
+       Statistic: Sum
+       Period: 600
+       EvaluationPeriods: 1
+       Threshold: 1
+       ComparisonOperator: GreaterThanOrEqualToThreshold
+      DependsOn: DigitalSubAuthLambda
```

Comment:
my intention with this 10 minute period and threshold of 1 is that we get don't get multiple emails for every single invocation is throttled but we get at the email when only one invocation is throttled.